### PR TITLE
Lb css cleanup

### PR DESCRIPTION
**Pull request by lolakoala**
Various small changes throughout the code to handle some minor UI details. 

I gave all buttons/links a border so they won't pulse on hover.
I made styling of all tan 'header' boxes consistent.
I toggle disabled on buttons in the join house component based on whether input fields have content.
I use color to signal whether bill is paid/chore is done.
I fix the tests that I broke while making these changes.

Good luck deciphering any of this in the code. Not sure where to tell you to look first, it's pretty scattered.

### CODE REVIEW

File: app/components/BillsList/BillsList.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ class BillsList extends Component {
         <Link to={`bills/${bill.id}`}>{bill.title}</Link>
         <p className='due'>{`Due: ${bill.duedate}`}</p>
         <p className='total'>{`My Total: ${user.total}`}</p>
-        <p className='mark-paid' onClick={() => this.markBillPaid(bill.id, currentUser.id, usersHouse)}>{user.paid ? `I'm paid up.` : 'Mark Paid'}</p>
+        <p className={`paid-${user.paid}`} onClick={() => this.markBillPaid(bill.id, currentUser.id, usersHouse)}>{user.paid ? `I'm paid up.` : 'Mark Paid'}</p>
```
Comment (jenPlusPlus) [inline app/components/BillsList/BillsList.js:90]: i like the change you made to conditionally style this shit. i may use this myself!

**Review by nicktu12 — APPROVED** [review]
These border changes could be refactored using variables in scss!

File: app/components/BillsList/BillsList.scss [hunk 2]
```diff
@@ -56,9 +56,9 @@
   text-align: center;
 }
 
-.bill .mark-paid {
+.bill .paid-false {
```
Comment (nicktu12) [inline app/components/BillsList/BillsList.scss:59]: This is a nice UX touch!

**Review by nicktu12 — APPROVED** [review]
you are the test master

**Review by jenPlusPlus — COMMENTED** [review]
i love the conditional classNames, like className='paid-${user.paid}' to determine how to style the element. nice use of booleans!

giphy 1

**Review by jenPlusPlus — APPROVED** [review]

**Pull request merged**